Patch to add Solaris 12 to the list of supported OSes.  This patch has
been submitted upstream.

--- puppetlabs-ntp-3.3.0/metadata.json.orig	2015-05-13 13:43:49.611523104 -0600
+++ puppetlabs-ntp-3.3.0/metadata.json	2015-05-13 13:44:30.682302981 -0600
@@ -66,7 +66,8 @@
     {
       "operatingsystem": "Solaris",
       "operatingsystemrelease": [
-        "11"
+        "11",
+        "12"
       ]
     },
     {
@@ -88,7 +89,7 @@
       "version_requirement": "3.x"
     }
   ],
-  "description": "NTP Module for Debian, Ubuntu, CentOS, RHEL, OEL, Fedora, FreeBSD, ArchLinux and Gentoo.",
+  "description": "NTP Module for Debian, Ubuntu, CentOS, RHEL, OEL, Fedora, FreeBSD, ArchLinux, Gentoo, and Solaris.",
   "dependencies": [
     {"name":"puppetlabs/stdlib","version_requirement":">= 3.2.0 < 5.0.0"}
   ]
